src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -214,7 +216,7 @@
                     '$ref': '#/definitions/fieldset',
                 },
             },
-            'required': ['required_fields'],
+            'required': ['type', 'required_fields'],
```

Comment:
This doesn't actually need to require `type` – this is a property within a `select` component. Only the top-level `select` part needs to have `type`.

Reply:
don't we need `'type': 'alert-rule-action'` ? Am i missing something?

Reply:
oh whoops, yea, sorry, the GitHub folding made this look like it was way up further in the file. My bad. This is right.